### ISSUE TITLE
Regarding background colour and transparency setting

### ISSUE THREAD

**Issue by torvista** [DivaVocals/zen_Image-Handler#18]
Regarding transparency and background colour.
I was getting this on a php7.1 dev server, while nitpicking: 
PHP Notice:  Undefined offset: 1 in blah-blah\includes\classes\bmz_image_handler.class.php on line 689
PHP Notice:  Undefined offset: 2 in blah-blah\includes\classes\bmz_image_handler.class.php on line 689

On IH install, the constants SMALL/MEDIUM/LARGE_BACKGROUND are all set to 255:255:255.
The text for these constants mentions using -transparent- (which is misleading in itself, no hyphens are required).

in bmz_image_hander_conf, a default setting is defined:
$ihConf['default']['bg']        = 'transparent 255:255:255';//

but suprisingly, this is not used when checking the constants in /functions_bmz_image_handler.php
$ihConf['small']['bg']          = defined('SMALL_IMAGE_BACKGROUND') ? SMALL_IMAGE_BACKGROUND : 'transparent 255:255:255';

instead a hard-coded value is used.

Anyway, when this background is looked at in bmz_image_handler_class.php: 
function get_background_rgb($bg)

		list($red, $green, $blue)= preg_split('/[, :]/', $bg);
		
there is no check if the parameters exist 123:123:123

I (for some unknown reason) had "transparent" in my admin settings and no colour parameters. Hence the original error.

So in conclusion, in my opinion:
- the descriptive text for the three constants needs improving with an example.
- the structure needs to be checked when the constants are checked, with a messagestack error if incorrect.
- the default $ihConf['default']['bg'] should be used where no constant is defined, instead of three hard coded parameters.
